front/app/components/FormBuilder/components/FormBuilderTopBar/index.tsx

```diff
@@ -59,13 +64,14 @@ const FormBuilderTopBar = ({
   }
 
   const isPostingEnabled = getIsPostingEnabled(project, phase);
-  const viewSurveyLInk = phaseId
+  const viewFormLink = phaseId
```

Comment:
Sure! Yeah good point - it's the same URL in this scenario (ideation and native survey) but it might not be for the next configuration we add after :+1: